cull_secrets/clean_user_registry_secrets.py

```diff
@@ -55,7 +61,13 @@ def find_pod_by_secret(secret, k8s_client):
 def remove_user_registry_secret(namespace, k8s_client, max_secret_age_hrs=0.25):
     """Used in a cronjob to periodically remove old user registry secrets"""
     secret_name_regex = ".+-registry-[a-z0-9-]{36}$"
-    label_keys = ["renku.io/commit-sha", "renku.io/projectName", "renku.io/username"]
+    label_keys = [
```

Comment:
You could define the `label_keys` at the module level and use it both locations. One day someone inattentive (like myself) will modify this list in only one of the two locations and produce a bug.